gbpservice/nfp/config_orchestrator/agent/loadbalancerv2.py

```diff
@@ -0,0 +1,457 @@
+        # Fetch nf_id from description of the resource
+        nf_id = self._fetch_nf_from_resource_desc(loadbalancer["description"])
+        nf = common.get_network_function_details(context, nf_id)
+        self._post(
+            context, healthmonitor['tenant_id'],
+            'healthmonitor', nf, healthmonitor=healthmonitor)
+
+    @log_helpers.log_method_call
+    def create_healthmonitor(self, context, healthmonitor):
+        loadbalancer = healthmonitor['pool']['loadbalancer']
+        # Fetch nf_id from description of the resource
+        nf_id = self._fetch_nf_from_resource_desc(loadbalancer["description"])
+        nf = common.get_network_function_details(context, nf_id)
+        self._post(
+            context, healthmonitor['tenant_id'],
+            'healthmonitor', nf, healthmonitor=healthmonitor)
+
+    @log_helpers.log_method_call
+    def delete_healthmonitor(self, context, healthmonitor):
+        loadbalancer = healthmonitor['pool']['loadbalancer']
+        # Fetch nf_id from description of the resource
+        nf_id = self._fetch_nf_from_resource_desc(loadbalancer["description"])
+        nf = common.get_network_function_details(context, nf_id)
+        self._delete(
+            context, healthmonitor['tenant_id'],
+            'healthmonitor', nf, healthmonitor=healthmonitor)
+
+    # TODO(jiahao): L7policy support not implemented
+    # disable L7policy
+    # def create_l7policy(self, context, l7policy):
+    #     self._post(
+    #         context, l7policy['tenant_id'],
+    #         'l7policy', l7policy=l7policy)
+    #
+    # def delete_l7policy(self, context, l7policy):
+    #     self._delete(
+    #         context, l7policy['tenant_id'],
+    #         'l7policy', l7policy=l7policy)
+    #
+    # def create_l7policy_rule(self, context, rule, l7policy_id):
+    #     self._post(
+    #         context, rule['tenant_id'],
+    #         'rule', rule=rule)
+    #
+    # def delete_l7policy_rule(self, context, rule):
+    #     self._delete(
+    #         context, rule['tenant_id'],
+    #         'rule', rule=rule)
+    #
+    # def _get_lb_context(self, context, filters):
+    #     args = {'context': context, 'filters': filters}
+    #     db_data = super(Lbv2Agent, self)
+    #     return {'loadbalancers': db_data.get_loadbalancers(**args),
+    #             'listeners': db_data.get_listeners(**args),
+    #             'pools': db_data.get_pools(**args),
+    #             'pool_members': db_data.get_pool_members(**args),
+    #             'healthmonitors': db_data.get_healthmonitors(**args),
+    #             'l7policies': db_data.get_l7policies(**args),
+    #             'l7policy_rules': db_data.get_l7policy_rules(**args)}
+
+
+class LoadbalancerV2Notifier(object):
+
+    def __init__(self, conf, sc):
+        self._sc = sc
+        self._conf = conf
+
+    # TODO(jiahao): copy from v1 agent need to review
+    def _prepare_request_data(self, context, nf_id,
```

Comment:
This function is used to send VIP deleted event to visibility in case of LBaaS V1. We need to check as part of which API, the vip port of LB is created in LBaaSV2 and send the event there. Check loadbalancer.py